A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_IdentPage.js

```diff
@@ -23,19 +23,19 @@ class CDUIdentPage {
         mcdu.page.Current = mcdu.page.IdentPage;
         mcdu.activeSystem = 'FMGC';
         mcdu.setTemplate([
-            ["A320-200"],
+            ["A320-251N"],
             ["\xa0ENG"],
-            ["LEAP 1A-26[color]green"],
+            ["LEAP-1A26[color]green"],
             ["\xa0ACTIVE NAV DATA BASE"],
-            ["\xa0" + (date.length === 13 ? date[3] + date[4] + date[0] + date[1] + date[2] + "-" + date[8] + date[9] + date[5] + date[6] + date[7] : date) + "[color]cyan", "AIRAC[color]green"],
+            ["\xa0" + (date.length === 13 ? date[3] + date[4] + date[0] + date[1] + date[2] + "-" + date[8] + date[9] + date[5] + date[6] + date[7] : date) + "[color]cyan", "FBW2014001[color]green"],
             ["\xa0SECOND NAV DATA BASE"],
-            ["{small}{04MAY-04JUL{end}[color]cyan"],
-            ["", "STORED"],
-            ["", "{green}10{end}{small}RTES\xa0{end}{green}10{end}{small}RWYS{end}"],
-            ["CHG CODE", "[b-text]{big}{green}20{end}{end}WPTS\xa0{big}{green}20{end}{end}NAVS"],
```

Comment:
Is there a reason you deleted this?
```suggestion
            ["", "{green}00{end}{small}RTES\xa0{end}{green}00{end}{small}RWYS{end}"],
             ["CHG CODE", "[b-text]{big}{green}00{end}{end}WPTS\xa0{big}{green}00{end}{end}NAVS"],
```
ref: 
<img width="452" alt="Screenshot 2021-02-03 at 16 19 35" src="https://user-images.githubusercontent.com/26572019/106776042-9db63600-663b-11eb-9432-babc006f548e.png">

Reply:
tarek said it was not supposed to be there so did komp

Reply:
lgtm, verified with another pilot just to make sure and check honeywell guide